lively.merger/README.md

```diff
@@ -1,3 +1,24 @@
 # lively.merger
 
-No description for package lively.merger yet.
+## Entry Point
+
+The entry point is the merger class. In this the different functionality for merging morphs is described.
+This falls back on the 3-way-merge class. In this the actual merge happens on object level.
+For a 3-way-merge we need 3 objects. These are compared on the basis of their keys. To merge morphs we use the propertiesAndPropertySettings method.
+So we merge only the properties of the morphs and then merge them to a new morph or overwrite the properties at one of the original morphs.
+
+## Custom Merging
+
+To make it easier for programmers to implement their own merge strategies there is the __provideMergeStrategy__ method. If this is implemented for an object which should be merged, this method is used instead of the standard merge algorithm.
+Over it programmers can decide very freely how exactly custom objects are to be merged, of course the 3-way-merge method is nevertheless available. Thus one can adapt also afterwards individual characteristics.
```

Comment:
Rephrasing and adding back ticks so that `__provideMergeStrategy__` is actually printed as such and not as __provideMergeStrategy__

```suggestion
To allow programmers to implement their own merge strategies, we added the `__provideMergeStrategy__` method. If this is implemented for an object that is about to be merged, this method is used instead of the standard merge algorithm. 

This allows for customization where the general purpose merge might ignore semantics. Colors for example are a collection of color channel values. If a general purpose merge would be applied, the colors themselves would not be merged, but their values, which would result in a mixed color and not the color of one of the versions.
```